string description
string arbitration_id
string data

